src/mame/includes/bublbobl.h

```diff
@@ -2,31 +2,43 @@
 // copyright-holders:Chris Moore, Nicola Salmoria
 
 #include "cpu/m6805/m68705.h"
-
+#include "cpu/m6800/m6801.h"
```

Comment:
If these aren't used in the header itself, they shouldn't be included from the header - it slows down parse and makes compilation slower.  I've been trying to cut down on this.